bundles_test.go

```diff
@@ -2100,7 +2219,7 @@ func Test_Bundles_Validate(t *testing.T) {
 					Deprecated:   false,
 					Name:         "kubernetes-operator",
 					Time:         time.Unix(10, 5),
-					Version:      "0.1.0",
+					Version:      "0.1.1",
```

Comment:
We now should fail only for bundles from the same series. (e.g. 0.1.1 and 0.1.0)

Reply:
we don't care if 0.0.9 was released after 0.1.0